src/main/java/com/minecolonies/coremod/inventory/api/IWorldNameableModifyable.java

```diff
@@ -0,0 +1,16 @@
+package com.minecolonies.coremod.inventory.api;
+
+import net.minecraft.world.IWorldNameable;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Created by marcf on 3/25/2017.
+ */
+public interface IWorldNameableModifyable extends IWorldNameable
+{
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)